SimG4CMS/Calo/interface/HcalTestAnalysis.h

```diff
@@ -62,36 +62,36 @@ class HcalTestAnalysis : public SimWatcher,
 private:
 
   //Keep parameters to instantiate HcalTestHistoClass later
-  std::string               fileName;
+  std::string                              fileName_;
 
   // Qie Analysis
-  HcalQie *                 myqie;
-  int                       addTower;
+  std::unique_ptr<HcalQie>                 myqie_;
+  int                                      addTower_;
 
   // Private Tuples
-  std::unique_ptr<HcalTestHistoManager>    tuplesManager;
-  HcalTestHistoClass        *tuples;
+  std::unique_ptr<HcalTestHistoManager>    tuplesManager_;
+  HcalTestHistoClass*                      tuples_;
 
   // Numbering scheme
-  HcalNumberingFromDDD      *numberingFromDDD;
-  HcalDDDSimConstants       *hcons;
-  HcalTestNumberingScheme   *org;
+  std::unique_ptr<HcalNumberingFromDDD>    numberingFromDDD_;
+  const HcalDDDSimConstants*               hcons_;
+  HcalTestNumberingScheme*                 org_;
 
   // Hits for qie analysis
-  std::vector<CaloHit>      caloHitCache; 
-  std::vector<int>          group_, tower_;
-  int                       nGroup, nTower;
+  std::vector<CaloHit>                     caloHitCache; 
```

Comment:
@bsunanda - please, rename all private members to have trailing underscore for consistency.